src/flint/types/acb.pyx

```diff
@@ -351,7 +351,7 @@ cdef class acb(flint_scalar):
         Complex sign function defined as a piecewise extension of
         the real sign function.
 
-            >>> from flint import showgood
+            >>> from flint.types.eval_helper import showgood
```

Comment:
The docstring should show how users are expected to import something.

Does `from flint import showgood` not work with these changes?

Reply:
I could import showgood as all the other types are so it's available from flint itself but I left it this way as my understanding it was more for debugging? 

The change should be minor I hope, happy for you to make the call

Reply:
It is shown widely in the docs so it should be treated as an end user function. Previously the docs always showed it as being imported from `flint` so we should preserve that for now (and continue to show that in the docs).

I think that `from flint import showgood` should continue to work and should be what is shown in the docstrings.

Reply:
Okay, I'm happy to make the changes to fix this (tomorrow, sleep now). Won't be more than a few extra lines of code :)